playbooks/roles/dev-patcher/tasks/main.yml

```diff
@@ -29,6 +29,17 @@
 
 - name: Get patch details
   include_tasks: patch-repos.yml
-  loop: "{{ dev_patcher_patches }}"
+  loop: "{{ dev_patcher_patches | default([], True) }}"
```

Comment:
dev_patcher_patches is always defined in defaults, no need to default it.

Reply:
I saw this error locally when we don't have any review added in list, then there is error reported. It just handles that case which can/will happen down the line.

Reply:
This is because you edited to remove something from the role. Assuming users don't edit the roles (because they never have to, nor they should), it should never be a problem.

It's defined by default, no need to redefine a default, that's what `defaults/main.yml` are for.

Reply:
Why we would not have case where there are no values defined for https://github.com/SUSE-Cloud/socok8s/blob/master/playbooks/roles/dev-patcher/defaults/main.yml#L10 . This can happen when we don't have any more upstream reviews to consume.

I had this case initially when mentioned patch was merged, so I removed it from value list and now list needed to be empty, so removed it and saw this error when executed the play. I did not know how to define empty list in yaml for that variable so ended removing it. So here we are trying to handle that case.

Reply:
If you have no values, you should do
``dev_patcher_patches: []`` , not comment things. Totally worth documenting though.